Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -0,0 +1,139 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class EditorialProjectsViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var closeButton: UIButton = {
+    UIButton(type: .custom)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var discoveryPageViewController: DiscoveryPageViewController = {
+    DiscoveryPageViewController.configuredWith(sort: .newest)
+      |> \.preferredBackgroundColor .~ .clear
+  }()
+
+  private lazy var headerView: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let viewModel: EditorialProjectsViewModelType = EditorialProjectsViewModel()
+
+  // MARK: - Lifecycle
+
+  public override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.configureSubviews()
+    self.setupConstraints()
+
+    self.closeButton.addTarget(
+      self,
+      action: #selector(EditorialProjectsViewController.closeButtonTapped),
+      for: .touchUpInside
+    )
+
+    // Will be moved to be a testable output of VC that presents this one
+    self.viewModel.inputs.configure(with: .goRewardless)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  public override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+
+    let currentTableViewInsets = self.discoveryPageViewController.tableView.contentInset
+
+    self.discoveryPageViewController.tableView.contentInset = currentTableViewInsets
+      |> UIEdgeInsets.lens.top .~ (self.headerView.frame.height - self.view.safeAreaInsets.top)
+
+    self.discoveryPageViewController.tableView.scrollIndicatorInsets =
+      self.discoveryPageViewController.tableView.contentInset
+  }
+
+  public override var preferredStatusBarStyle: UIStatusBarStyle {
+    return .lightContent
```

Comment:
This works for the dark blue on the header but not so much when we scroll the cards up 🤷‍♂